tests/containers/privileged_mode.pm

```diff
@@ -25,8 +26,9 @@ sub run {
     my $image = "registry.suse.com/bci/bci-base:latest";
 
     record_info('Test', 'Launch a container with privileged mode');
+
     # /dev is only accessible in privileged mode
-    assert_script_run("$runtime run --rm --privileged $image ls /dev/bus");
+    assert_script_run("$runtime run --rm --privileged $image ls /dev/bus") unless is_s390x;
```

Comment:
Ok for fixing the test runs, however now we only have one test for the privileged mode, which is not enough. We need to extend this further.

Reply:
I added the requirement to the ticket